### PR TITLE
PP9-10478 - Samples: Show Version number for Apps

### PR DESCRIPTION
**Pull request by tc4rdpp**
- Content Portal
 - Press Portal

### CODE REVIEW

File: Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/appsettings.json [hunk 1]
```diff
@@ -67,5 +67,9 @@
                 "Url": "http://localhost:5000"
             }
         }
+    },
+    "AppInfo": {
+        "Name": "Content Portal",
+        "Version": "10.6.0"
```
Comment (neotek) [inline Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/appsettings.json:73]: Read Product version from executing Assembly.

File: Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/Controllers/AppController.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Mvc;
```
Comment (neotek) [inline Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/Controllers/AppController.cs:1]: Would add this to existing configuration controller to save an additional http request.

File: Picturepark.PressPortal.Demo/Picturepark.PressPortal.Demo/appsettings.json [hunk 3]
```diff
@@ -38,5 +38,8 @@
     },
     "Authorization": {
         "AutoAssignUserRoleIds": []
-    }
+  },
+  "AppInfo": {
+     "Version": "10.6.0"
```
Comment (neotek) [inline Picturepark.PressPortal.Demo/Picturepark.PressPortal.Demo/appsettings.json:43]: Read Product version from executing Assembly

Comment (tc4rdpp) [reply Picturepark.PressPortal.Demo/Picturepark.PressPortal.Demo/appsettings.json]: Done

File: Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/Controllers/AppController.cs (hunk 2 above)
Comment (tc4rdpp) [reply Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/Controllers/AppController.cs]: Done

File: Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/appsettings.json (hunk 1 above)
Comment (tc4rdpp) [reply Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/appsettings.json]: Done

File: Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/Controllers/ConfigurationController.cs [hunk 4]
```diff
@@ -18,13 +20,15 @@ public ConfigurationController(IConfiguration configuration)
         public IActionResult GetClientConfiguration()
         {
             var config = _configuration.GetSection("PictureparkConfiguration").Get<PictureparkConfiguration>();
+            var version = FileVersionInfo.GetVersionInfo(Assembly.GetEntryAssembly().Location).ProductVersion; ;
```
Comment (tc4rdpp) [inline Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/Controllers/ConfigurationController.cs:23]: remove semicolon

**Pull request merged**